src/components/buttons/button.tsx

```diff
@@ -1,29 +1,45 @@
 import React from 'react';
 import styled from 'styled-components';
-import { TextStyles } from '../typography/typography-v2';
+import { Icon, IconSize } from '../icon/icon';
+import { resetButton } from '../utils';
+import { lighten } from 'polished';
 
 const StyledButton = styled.button`
-  height: 40px;
+  ${resetButton};
+  color: #ffffff;
+  font-size: 16px;
+  font-weight: bold;
+  line-height: 40px;
+  border-radius: 30px;
+  vertical-align: middle;
+  cursor: pointer;
 
   background: linear-gradient(275.41deg, #543fd7 0%, #2756fd 100%);
-  border-radius: 30px;
-  border: none;
 
   &:hover {
     background: #3e61ee;
   }
+
+  &:active {
+    background: ${lighten(0.1, '#3e61ee')};
```

Comment:
This lighten helper from polished is my simple way of defining some lighter color here. That `active` state (when mouse is down on the button) is not defined in the design and I don't want to postpone every single small decision until the design team delivers. Here a best guess to be fixed and aligned later 👍

Reply:
I'd like to point out that [SCSS also provides this](https://sass-lang.com/documentation/modules/color#scale). No need for an additional library.
I know that we already have `polished` as a dependency so might as well use it. I just think it's worth knowing that SCSS has an insane amount of functionality built in.

Reply:
That's irritating first but styled components does not provide scss ☝️ It supports a small set of feature like the nesting only. That's why polished is offered by the maintainers of styled components.

Reply:
Thanks for the info. This is in fact irritating... We don't need new standards at every corner 😭